src/main/java/com/speaker/service/AccountServiceImpl.java

```diff
@@ -65,6 +75,15 @@ public Response create(AccountDTO accountDTO) {
         return Response.FALSE;
     }
 
+
+    private Optional<Integer> convertFirstAndLastNameToId(String names) {
+        String[] splitNames = names.split("\\s");
```

Comment:
if I type names with double space will be works?

Reply:
Thanks for the comment. Error fixed, correction made.